.github/workflows/pull-request.yml

```diff
@@ -298,3 +298,9 @@ jobs:
           MAVEN_OPTS: "-Xmx2g"
       - name: 'Clean Maven cache'
         run: rm -rf ~/.m2/repository/org/eclipse/collections
+
+  forbid-merge-commits:
```

Comment:
I am not entirely sure if it will immediately be evident to the contributor what their next steps are if this action fails - is it possible to include a section regarding the rebase steps in the contributing guidelines?

Reply:
When the check fails, it leaves instructions on how to fix the problem. I can't demonstrate it on this repository until it lands, but I could demonstrate it in another repository if that would be helpful.